server/controller/src/main/java/ai/starwhale/mlops/api/DatasetApi.java

```diff
@@ -41,8 +41,14 @@
         description =
             "Select a historical version of the dataset and revert the latest version of the current dataset to this version")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "ok")})
-    @PostMapping(value = "/dataset/{datasetId}/revert")
+    @PostMapping(value = "/project/{projectId}/dataset/{datasetId}/revert")
```

Comment:
Does the dataset belong to one project?  why are datasets not at the same level as projects?

Reply:
Yes.  Each project contains several models datasets and jobs

Reply:
got